### PR TITLE
use shell builtin `command` instead of `which`

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: etc/condor/scripts/worker.W2012Pipe.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ module add gcc/4.4.5
 
 echo ${HERE}
 
-which gcc
+command -v gcc
```
Comment (timj) [inline etc/condor/scripts/worker.W2012Pipe.sh]: This is just telling the user what compiler is in the path??? Maybe in that case `type -a gcc` would be more informative?

Comment (jhoblitt) [reply etc/condor/scripts/worker.W2012Pipe.sh]: I assumed the intent was to document the actual compiler used.

Comment (timj) [reply etc/condor/scripts/worker.W2012Pipe.sh]: Sure, but this seems clearer:
```
$ type -a clang
clang is /usr/bin/clang
```
than
```
$ command -v clang
/usr/bin/clang
```

Comment (jhoblitt) [reply etc/condor/scripts/worker.W2012Pipe.sh]: That's reasonable.

**Comment by jhoblitt** [comment]
I'm going to go ahead and merge this since it is trivial.

**Pull request merged**